### ISSUE TITLE
add some feature

### ISSUE THREAD

**Issue by harry-maulana** [noam148/yii2-image-manager#11]
hi Noam..

how to add some feature, like as rotate, zoomin, etc.
thanks

**Comment by noam148** [comment]
Sounds good, I will work on it. But first I want to add a feature that make possible to add tag to images for categorize them. I let you know when to start with you issue.

**Comment by abdallah-elsabeeh** [comment]
hi how to get the image URL to store in the DB

**Comment by harry-maulana** [comment]
hi Noam, thx for reply.
can you let we all know, how this plugin (jquery crop) work.. in one layer for all action/feature or need one special layer for each action/feature. like as in my case, i want to add rotate feature, whether i should add new layer?
thx before..

**Comment by harry-maulana** [comment]
@zaxx44a7 i don't know too zaxx, I just know there are plugins as good like this.

**Comment by noam148** [comment]
@zaxx44a7 If you get the ImageManager_id you can generate a cache url what is possible to store in your database by call this component:

`
\Yii::$app->imagemanager->getImagePath($ImageManager_id, $width, $height, $thumbnailMode)
`

**Comment by noam148** [comment]
@harry-promogata I use this jquery plugin to crop the image [https://fengyuanchen.github.io/cropper/](url). So as you can see in the example there is possibilities for rotating and flip the image. The zoom options is already in the module.

**Comment by harry-maulana** [comment]
yes. that is possible, but i don't know how to add that feature.. lol,
so i wait your job next (if you want to add thats feature)

thx Noam..

**Comment by noam148** [comment]
@harry-promogata yes I wil add them, I let you know if the new feature are add!

**Comment by ghost** [comment]
Hi, can there be different image upload locations?
